File: app/src/main/java/com/slimshady/bottomsheetmodule/adapter/listener/AdapterItemMultiSelectListener.java
package com.slimshady.bottomsheetmodule.adapter.listener;

import com.slimshady.bottomsheetmodule.util.AdapterAction;

import java.util.ArrayList;

public interface AdapterItemMultiSelectListener<T> {
    void onItemMultiSelect(ArrayList<T> model, AdapterAction action);

}

File: app/src/main/java/com/slimshady/bottomsheetmodule/adapter/listener/AdapterItemMultiSelectListener.kt
package com.slimshady.bottomsheetmodule.adapter.listener

import com.slimshady.bottomsheetmodule.util.AdapterAction
import java.util.ArrayList

interface AdapterItemMultiSelectListener<T> {
    fun onItemMultiSelect(model: ArrayList<T>?, action: AdapterAction?)
}